docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md

```diff
@@ -1,22 +1,89 @@
 ---
-title: Dynamic Multistore feature overview
-description: Dynamic Multistore lets you create and manage multiple online stores in the Back Office.
+title: Dynamic Multistore
+description: Dynamic Multistore lets you create and manage multiple online stores from the Back Office.
 last_updated: July 31, 2023
 template: concept-topic-template
 ---
 
-{% info_block warningBox %}
+The *Dynamic Multistore* feature lets you create and manage multiple online stores within the same region of the Back Office installation. It streamlines the setup and maintenance of distinct stores tailored to various customer segments, regions, or product categories. 
 
-Dynamic Multistore is part of an *Early Access Release*. This *Early Access* feature introduces the ability to handle the store entity in the Back Office. Business users can try creating stores without editing the `Stores.php` file and redeploying the system.
+In the following sections, you can learn more details about the capabilities of Dynamic Multistore.
```

Comment:
What value does this sentence give?

Reply:
It just seemed right to add an intro sentence for the sections